configure.in

```diff
@@ -0,0 +1,25 @@
+#                                               -*- Autoconf -*-
+# Process this file with autoconf to produce a configure script.
+
+AC_PREREQ([2.69])
+AC_INIT(AVLDupTree, 1.0.0, https://github.com/HaikuArchives/AVLDupTree/issues)
+AC_CONFIG_SRCDIR([Source/AVLDupTree.c])
+# AC_CONFIG_HEADERS([config.h])
+
+# Checks for programs.
+AC_PROG_CXX
```

Comment:
AVLDupTree isn't an C++ project and doesn't use the C++ compiler. (CXX == C++ compiler)